firmware/src/gem_settings_load_save.c

```diff
@@ -94,7 +105,7 @@ bool GemSettings_load(struct GemSettings* settings) {
 
     uint8_t marker = data[0];
 
-    if (marker != SETTINGS_MARKER_V1 && marker != SETTINGS_MARKER_V2 && marker != SETTINGS_MARKER_V3) {
+    if (marker < SETTINGS_MARKER_MIN || marker > SETTINGS_MARKER_MAX) {
```

Comment:
Nice

Reply:
Thank you! I figured it might be good to tidy up a little while I'm here :)